internal/usecases/pegin/get_pegin_report_test.go

```diff
@@ -0,0 +1,96 @@
+package pegin_test
+
+import (
+	"context"
+	"github.com/rsksmart/liquidity-provider-server/internal/entities"
+	"github.com/rsksmart/liquidity-provider-server/internal/entities/quote"
+	"github.com/rsksmart/liquidity-provider-server/internal/usecases/pegin"
+	"github.com/rsksmart/liquidity-provider-server/test/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestGetPeginReportUseCase_Run(t *testing.T) {
+	ctx := context.Background()
+
+	retainedQuotes := []quote.RetainedPeginQuote{
+		{QuoteHash: "hash1"},
+		{QuoteHash: "hash2"},
+	}
+
+	peginQuotes := []quote.PeginQuote{
```

Comment:
please extend this dataset, having only 2 elements for a functionality that calculates min, max, avg, etc. Is too few

Reply:
done